charts/aws-ebs-csi-driver/templates/node-windows.yaml

```diff
@@ -108,7 +108,7 @@ spec:
           args:
             - --csi-address=$(ADDRESS)
             - --kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)
-            - --v=5
+            - --v={{ .Values.nodeWindows.logLevel }}
```

Comment:
The windows node currently uses the `node.*` values so this should be changed to match